code/modules/projectiles/boxes_magazines/external/pistol.dm

```diff
@@ -83,3 +83,13 @@
 	caliber = ".50"
 	max_ammo = 7
 	multiple_sprites = AMMO_BOX_PER_BULLET
+
+/obj/item/ammo_box/magazine/liberator10mm
+	name = "Makeshift 9mm magazine"
+	desc = "A Makeshift Pistol magazine. Load with 9mm bullets. Usable in Makeshift Pistol."
+	icon_state = "libmag"
+	ammo_type = /obj/item/ammo_casing/c9mm

```

Comment:
are you sure you want people to be doing 30 damage? You could kill someone in less than 2 seconds from across the screen if you just made two of these

Reply:
True, but at those distances your target is going to have to not be whizzing around since bullets in SS13 arent exactly faster than sound

Reply:
My suggestion is that since it's a ghetto pistol make it either very inacurate with high spread, or have a chance of backfire like a tinkered det's revolver.